draft-ietf-openpgp-pqc.md

```diff
@@ -564,36 +547,31 @@ The ML-KEM + ECDH composite public-key encryption schemes are built according to
 ### Key combiner {#kem-key-combiner}
 
 For the composite KEM schemes defined in {{kem-alg-specs}} the following procedure MUST be used to compute the KEK that wraps a session key.
-The construction is a one-step key derivation function compliant to {{SP800-56C}}, Section 4, based on KMAC256.
+The construction is a key derivation function compliant to {{SP800-108r1}}, Section 4.4, based on KMAC256.
 It is given by the following algorithm, which computes the key encryption key `KEK` that is used to wrap, i.e., encrypt, the session key.
 
-\[Note to the reader: the key combiner defined in the current version of this draft is not actually compliant to {{SP800-56C}}, since the NIST standard requires that the shared secret is fed to the KDF first whereas the combiner defined here feeds
-the key shares of the two component schemes, which together form the shared secret, in two parts with public information in between.
-The combiner will be reworked to fix this defect.
-The change is planned to be integrated prior to IETF 121.\]
-
 
-    //   multiKeyCombine(ecdhKeyShare, ecdhCipherText, ecdhPublicKey, mlkemKeyShare,
-    //                   mlkemCipherText, mlkemPublicKey, algId)
+    //   multiKeyCombine(mlkemKeyShare, mlkemCipherText, mlkemPublicKey, ecdhKeyShare,
+    //                   ecdhCipherText, ecdhPublicKey, algId)
```

Comment:
What still needs to be done is to adapt the invocations of `multiKeyCombine()` to the new order or arguments.